### ISSUE TITLE
Importing components trigger eslint rules

### ISSUE THREAD

**Issue by nseptier** [sveltejs/eslint-plugin-svelte3#86]
Hi.

Eslint reports unused vars/missing declarations when I try to import and use components in my svelte files. I can't figure out what's wrong.

```svelte
<script lang="ts">
  import Navigation from './navigation/Navigation.svelte';
</script>

<Navigation />
```
```
  2:10  warning  'Navigation' is defined but never used  @typescript-eslint/no-unused-vars
  5:1   warning  'Navigation' is not defined             missing-declaration
```

Here's my config file:
```
module.exports = {
  parser: '@typescript-eslint/parser',
  parserOptions: {
    extraFileExtensions: ['.svelte'],
    project: ['./tsconfig.json'],
    sourceType: 'module',
    tsconfigRootDir: __dirname,
  },
  overrides: [
    {
      files: ['*.svelte'],
      processor: 'svelte3/svelte3',
    },
  ],
  plugins: ['svelte3', '@typescript-eslint'],
  settings: {
    'svelte3/typescript': require('typescript'),
  },
  env: {
    browser: true,
    es2021: true,
  },
  extends: [
    'eslint:recommended',
    'plugin:@typescript-eslint/recommended',
    'plugin:@typescript-eslint/recommended-requiring-type-checking',
  ],
  rules: {},
}
```

**Comment by dummdidumm** [comment]
Should get fixed by #84

**Comment by Conduitry** [comment]
#84 has been released in 3.1.1.
